### PR TITLE
Update WHSSDatlas_v1.01.jsonld

### PR DESCRIPTION
**Pull request by aeidi89**

### CODE REVIEW

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld [hunk 1]
```diff
@@ -15,8 +15,8 @@
   "copyright": null,
   "custodian": null,
   "description": null,
-  "digitalIdentifier": null,
-  "fullDocumentation": null,
+  "digitalIdentifier": "https://scicrunch.org/resolver/RRID:SCR_017124",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: this has to be:
```
"digitalIdentifier": {"@id": "https://scicrunch.org/resolver/RRID:SCR_017124"},
```

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld [hunk 2]
```diff
@@ -254,18 +254,18 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Papp et al. 2015.",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: The references should best stated as full citations. you don't want people to search for them themselves (at least you have to state the DOI)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld [hunk 3]
```diff
@@ -254,18 +254,18 @@
...
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Papp et al. 2015.",
   "isAlternativeVersionOf": null,
   "isNewVersionOf": null,
   "keyword": null,
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld:263]: was the previous one registered incorrect or did you change it?

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld [hunk 4]
```diff
@@ -254,18 +254,18 @@
...
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": "https://doi.org/10.1016/j.neuroimage.2014.04.001",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: this has to be:
```
"relatedPublication": [
  {"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"}
],
```

**Review by lzehl — CHANGES_REQUESTED** [review]

**Review by lzehl — CHANGES_REQUESTED** [review]
sorry... for the erroneous review (forgot that digital identifier takes only single values)

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld (hunk 1 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: Fixed.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld (hunk 2 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: Full citations have been added.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld (hunk 4 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: Fixed.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld (hunk 3 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld:263]: We changed it on June 28, 2023.

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld [hunk 5]
```diff
@@ -254,18 +254,20 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v1.01 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2015) Addendum to "Waxholm Space atlas of the Sprague Dawley rat brain" [NeuroImage 97 (2014) 374-386]. NeuroImage 105, 561–562. DOI: 10.1016/j.neuroimage.2014.10.017).",
```
Comment (lzehl) [inline instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: @aeidi89 thanks. I suggest to define this as rst-compatible markdown:

`"Please refer to the atlas by its RRID: SCR_017124, and cite the following publications:\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: https://doi.org/10.1016/j.neuroimage.2014.04.001\n- Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2015) Addendum to "Waxholm Space atlas of the Sprague Dawley rat brain" [NeuroImage 97 (2014) 374-386]. NeuroImage 105, 561–562. DOI: https://doi.org/10.1016/j.neuroimage.2014.10.017"`

(\n means newline)

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld (hunk 5 above)
Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: Thank you @lzehl
Fixed now.

Comment (aeidi89) [reply instances/atlas/brainAtlasVersion/WHSSDatlas_v1.01.jsonld]: Thanks @lzehl 
Fixed now.

**Comment by aeidi89** [comment]
@lzehl Thank you for being patience with me and teaching me along the way.
I hope all is correct now.

**Review by lzehl — APPROVED** [review]
@aeidi89 you are very welcome and thanks for improving the registration of the atlases

**Pull request merged**